tests/core/test_open_in_process.py

```diff
@@ -3,17 +3,27 @@
 import signal
 
 import pytest
+import trio
 
 from asyncio_run_in_process import (
     ProcessKilled,
     constants,
     open_in_process,
+    open_in_process_with_trio,
 )
 from asyncio_run_in_process.exceptions import (
     ChildCancelled,
 )
 
 
+@pytest.fixture(params=('use_trio', 'use_asyncio'))
+def open_in_proc_and_sleep_fn(request):
+    if request.param == 'use_trio':
+        return open_in_process_with_trio, trio.sleep
```

Comment:
it might be cleaner to do something like this using [`sniffio`](https://github.com/python-trio/sniffio)

```python
# asyncio_run_in_process/tools/sleep.py
from sniffio import current_async_library

async def sleep(duration: float) -> None:
    if current_async_library() == 'trio':
        import trio
        await trio.sleep()
    elif current_async_library() == 'asyncio':
        import asyncio
        await asyncio.sleep()
    else:
        raise Exception("Invariant")
```

Reply:
Yeah, that's nice!